rundeckapp/test/unit/rundeck/quartzjobs/ExecutionJobSpec.groovy

```diff
@@ -169,4 +169,62 @@ class ExecutionJobSpec extends Specification {
 
 
     }
+    def "scheduled job was stoped by another cluster node, so should be deleted from quartz scheduler"() {
+        given:
+        def serverUUID = UUID.randomUUID().toString()
+        def jobUUID = UUID.randomUUID().toString()
+        def es = Mock(ExecutionService)
+        def eus = Mock(ExecutionUtilService)
+        def fs = Mock(FrameworkService) {
+            getServerUUID() >> serverUUID
+        }
+        ScheduledExecution se = new ScheduledExecution(
+                jobName: 'blue',
+                project: 'AProject',
+                groupPath: 'some/where',
+                description: 'a job',
+                argString: '-a b -c d',
+                serverNodeUUID: serverUUID,
+                workflow: new Workflow(
+                        keepgoing: true,
+                        commands: [new CommandExec(
+                                [adhocRemoteString: 'test buddy', argString: '-delay 12 -monkey cheese -particle']
+                        )]
+                ),
+                scheduled: false,
```

Comment:
You can parameterize the `scheduled`, `executionEnabled` and `scheduleEnabled` values to test all of them being set to false